### PR TITLE
Comments on Geospatial Publishing

### PR DESCRIPTION
**Pull request by codynhat**

### CODE REVIEW

File: docs/005-developers/008-geospatial-publishing/001-mud-world.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Geospatial content anchored on the Geo Web is published onchain to a single [MUD](https://mud.dev) World. MUD is an EVM framework that handles access control, indexing, and extensibility to enable building [Autonomous Worlds](https://0xparc.org/blog/autonomous-worlds).
+
+Each [PCO](../../concepts/partial-common-ownership) parcel in the Geo Web is assigned a namespace in the global MUD world. The current owner of the parcel is the only one who can take ownership of the corresponding namespace for that parcel. They are then free to do what they wish so long as they keep paying the network fee. This includes keeping write permissions to themselves, opening up writing to anyone or any combination in between.
+
+![](./mud-world.excalidraw.svg)
+
+## Global Hooks and Systems
+
+In order to enforce the mechanics of [PCO](../../concepts/partial-common-ownership), the world has a custom hook and system installed in the `/world` namespace.
```
Comment (gravenp) [inline docs/005-developers/008-geospatial-publishing/001-mud-world.md:11]: Link "custom hook" to https://mud.dev/store/advanced-features#storage-hooks ?

Comment (gravenp) [reply docs/005-developers/008-geospatial-publishing/001-mud-world.md:11]: This was the best link for "systems" that I could find: https://mud.dev/quick-start#systems

File: docs/005-developers/008-geospatial-publishing/002-augments.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Developing Augments
+
+An **Augment** is a MUD [module](https://mud.dev/world/modules) that can be installed onto a specific parcel in the [PCO World](./mud-world). A module is an onchain script that can pre-populate tables with content and install infrastructure like systems, hooks, or new tables. All Augments must conform to a simple onchain interface that enables them to be discovered and installed in a general way without any additional work needed by clients.
```
Comment (gravenp) [inline docs/005-developers/008-geospatial-publishing/002-augments.md:3]: Is it helpful to pull in some references to ECS here or in the MUD doc? I see in their docs (https://mud.dev/ecs) that MUDv2 isn't strictly ECS, but not sure if the mental model is still useful for devs.

Comment (codynhat) [reply docs/005-developers/008-geospatial-publishing/002-augments.md:3]: Maybe we should describe the design of tables somewhere? And can mention ECS there.

**Pull request merged**